README.md

```diff
@@ -1,8 +1,6 @@
-# curriculum-databases-projects-template
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
 
+This is an educational project.
```

Comment:
- [ ]  Kindly add the project name to your README file. Don't forget, we can describe a README file as a guide that gives users a detailed description of a project you have worked on. It can also be described as documentation with guidelines on how to use a project.

Reply:
Oh! I added the title and it seems by mistake I removed it! I remember it clearly but... :D Thanks for noticing it.